src/shared/components/palette/collection/write-collection-field.component.spec.ts

```diff
@@ -269,8 +269,6 @@ describe('WriteCollectionFieldComponent', () => {
     let writeFields = de.queryAll($WRITE_FIELDS);
     expect(writeFields.length).toBe(VALUES.length - 1);
 
-    let field2 = writeFields[0].componentInstance;
```

Comment:
After discussion with @Mo-Lala84 it was agreed that this would be removed. Further information can be found on the corresponding Jira ticket.